src/React/Basic/Hooks.purs

```diff
@@ -65,14 +56,10 @@ import Effect.Uncurried (EffectFn1, EffectFn2, EffectFn3, mkEffectFn1, runEffect
 import Prelude (bind) as Prelude
 import Prim.Row (class Lacks)
 import React.Basic (JSX, ReactComponent, ReactContext, Ref, consumer, contextConsumer, contextProvider, createContext, element, elementKeyed, empty, keyed, fragment, provider)
-import Type.Equality (class TypeEquals)
+import React.Basic.Hooks.Internal (Hook, Pure, Render, bind, discard, newtypeHook, unsafeHook, unsafeRenderEffect)
 import Unsafe.Coerce (unsafeCoerce)
 import Unsafe.Reference (unsafeRefEq)
 
--- | Alias for convenience.
-type CreateComponent props
-  = Effect (ReactComponent props)
```

Comment:
I don't think this alias is worth it cluttering the module docs and adding terms/concepts to learn